splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/BaseFormView.jsx

```diff
@@ -397,7 +402,7 @@ class BaseFormView extends PureComponent {
             if (this.props.mode === MODE_EDIT) {
                 if (this.hookDeferred) {
                     this.hookDeferred.then(() => {
-                        if (typeof this.hook.onCreate === 'function') {
+                        if (typeof this.hook.onEditLoad === 'function') {
```

Comment:
Any reason we are naming this `onEditLoad`? Is `onCreate` still available?

Reply:
That was by mistake Actually, We've two separate methods,